### PR TITLE
New feature for the Payload Inspector: support for two tags/ single IOV plot generation

### PR DESCRIPTION
**Pull request by ggovi**
In the payload Inspector application, we are adding the support of two tags for single iov payload comparison.

The same branch used for the PR has been deployed as the DEMO release in the Payload Inspector.  A dedicated new plot based on the BasicPayload class has been created.

### CODE REVIEW

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
+1

File: CondCore/Utilities/scripts/getPayloadData.py [hunk 1]
```diff
@@ -169,15 +210,17 @@ def discover():
...
+            result.setdefault(payload_type, []).append({'plot': plot, 'plugin_name': plugin_name, 'title': plot_title, 'plot_type': plot_type, 'single_iov': single_iov, 'two_tags': two_tags})
             output('currently discovered info: ', result)
     output('*** final output:', '')
     return json.dumps(result)
 
 def output(description, param):
     if args.verbose:
-        print('')
-        print(description, param)
+        print ''
```
Comment (mmusich) [inline CondCore/Utilities/scripts/getPayloadData.py]: isn't this going back with the python3 changes?

Comment (fabiocos) [reply CondCore/Utilities/scripts/getPayloadData.py]: @mmusich @ggovi yes, this will fail the python3 compiler check,....

**Comment by fabiocos** [comment]
+1

File: CondCore/Utilities/scripts/getPayloadData.py (hunk 1 above)
Comment (mmusich) [reply CondCore/Utilities/scripts/getPayloadData.py]: https://github.com/cms-sw/cmssw/pull/26484 addresses the problem created by this PR.

**Pull request merged**